SampleGetApi/ReportsHelper/ReportUtils.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Xml.Linq;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+
+namespace SampleGetApi.ReportsHelper;
+
+	public class ReportUtils
+	{
+    public static ExtentReports? ExtentReport;

```

Comment:
Not a fan of static. Any reason this is static?

Reply:
To make sure the report object is same through out the suite execution. Changing to non static will create unwanted objects to access the reference. 
2 different classes are accessing this reference, hook and basetest